### PR TITLE
refactor: new styles based on demo specs

### PR DESCRIPTION
**Pull request by kima-bastian**
Introduced some minor style changes from rebranded demo and feedback from Dan.

### CODE REVIEW

File: src/components/reusable/SingleForm.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ const SingleForm = ({}) => {
         <span className='label'>Source Network:</span>
         <div className='items'>
           <SourceNetworkSelector />
-          <SourceTokenSelector />
```
Comment (mayjer-kima) [inline src/components/reusable/SingleForm.tsx:93]: If these components are no longer needed then please remove them:
* SourceTokenSelector
* TokenBadge

File: src/components/reusable/ConfirmDetails.tsx [hunk 2]
```diff
@@ -147,31 +147,33 @@ const ConfirmDetails = ({ isApproved }: { isApproved: boolean }) => {
...
-              → <TargetCoinIcon /> {targetCurrency}
+              <p>
+              {formatterFloat.format(parseFloat(amountToShow)-totalFeeUsd)} {sourceCurrency}
+              </p>
             </div>
-          )}
+            {sourceCurrency !== targetCurrency && (
+              <div className='coin-details'>→ {targetCurrency}</div>
+            )}
+          </div>
```
Comment (mayjer-kima) [inline src/components/reusable/ConfirmDetails.tsx:166]: The spacing here makes the alignment of the "Transferring..." text  look off. It's centered within it's area but not centered relative to the progress bar or the status box.

image

**Review by mayjer-kima — COMMENTED** [review]
Mobile minimized mode looks ok (ignore the alignment outside the widgets that's the app styling)
image
image

File: src/components/reusable/ConfirmDetails.tsx (hunk 2 above)
Comment (kima-bastian) [reply src/components/reusable/ConfirmDetails.tsx:166]: I don't see the same, in my case it looks perfect D:

**Comment by mayjer-kima** [comment]
The `<CoinDropdown>` component does not use the `<TokenIcon>` component like `<SourceNetworkSelector>` did. There are multiple places where the icon mappings are defined still. So while the Base icon works with the `<TokenIcon>` component, it breaks in `<ConfirmDetails>` as Base wasn't added to the mapping there. We should consolidate it all into one place.

**Comment by kima-bastian** [comment]
Applying more changes, please don't merge yet

**Comment by kima-bastian** [comment]
Merging...

**Pull request merged**